content/applications/inventory_and_mrp/manufacturing/management/configure_manufacturing_product.rst

```diff
@@ -0,0 +1,67 @@
+===================================
+Configure product for manufacturing
+===================================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+In order to manufacture a product in Odoo *Manufacturing*, the product must be properly configured.
+Doing so consists of enabling the Manufacturing route and configuring a Bill of Materials (BoM) for
+the product. Once these steps are completed, the product is selectable when creating a new
+manufacturing order.
+
+Activate the Manufacture route
+==============================
+
+The Manufacture route is activated for each product on its product page. To do so, begin by
+navigating to :menuselection:`Manufacturing --> Products --> Products`, then select an existing
+product or create a new one by clicking :guilabel:`New`.
+
+On the product page, select the :guilabel:`Inventory` tab, then enable the :guilabel:`Manufacture`
+checkbox in the :guilabel:`Routes` section. This tells Odoo that the product can be manufactured.
+
+.. image:: configure_manufacturing_product/manufacturing-route.png
+   :align: center
+   :alt: The Manufacturing route on the Inventory tab of a product page.
+
+Configure a Bill of Materials (BoM)
+===================================
+
+Next, a |BOM| must be configured for the product so that Odoo will know how it is manufactured. A
+|BOM| is a list of the components and operations required to manufacture a product.
+
+To create a |BOM| for a specific product, navigate to :menuselection:`Manufacturing --> Products -->
+Products`, then select the product. On the product page, click the :guilabel:`Bill of Materials`
+smart button at the top of the page, then select :guilabel:`New` to configure a new |BOM|.
+
+.. image:: configure_manufacturing_product/bom-smart-button.png
+   :align: center
+   :alt: The Bill of Materials smart button on a product page.
+
+On the |BOM|, the :guilabel:`Product` field will auto-populate with the product. In the
+:guilabel:`Quantity` field, specify the number of units that the BoM produces.
+
+Add a component to the |BOM| by selecting the :guilabel:`Components` tab and clicking :guilabel:`Add
+a line`. Select a component from the :guilabel:`Component` drop-down menu, then enter the quantity
+in the :guilabel:`Quantity` field. Continue adding components on new lines until all components have
+been added.
+
+.. image:: configure_manufacturing_product/components-tab.png
+   :align: center
+   :alt: The Components tab on a bill of materials.
+
+Next, select the :guilabel:`Operations` tab. Click :guilabel:`Add a line` and a :guilabel:`Create
```

Comment:
The Operations tab on the BOM isn't available by default. We have to enable the Work Orders setting first. Perhaps add an admonition block following this section explaining how to do so?